azure-pipelines.yml

```diff
@@ -22,30 +22,19 @@ variables:
   workspaceDir: knotx-repos
 
 steps:
-  - task: Gradle@2
```

Comment:
This should be here.

Reply:
Ok, reverted.
Maybe we should use `development/build-stack.sh` in order to always build plugins first?